src/main/java/org/reactome/release/cosmicupdate/Main.java

```diff
@@ -0,0 +1,337 @@
+package org.reactome.release.cosmicupdate;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.sql.SQLException;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.stream.Collectors;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.reactome.release.common.ReleaseStep;
+import org.reactome.release.common.dataretrieval.cosmic.COSMICFileRetriever;
+import org.reactome.util.general.GUnzipCallable;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+
+public class Main extends ReleaseStep
+{
+	@Parameter(names = {"-u"}, description = "Specifies that the updates should be performed.")
+	private boolean executeUpdate;
+	
+	@Parameter(names = {"-d"}, converter = DurationConverter.class,
+			description = "Download the files, if the age of local files exceeds the specified max age, or if the files don't exist."
+					+ "Omitting this parameter means download will not occur. Specifying -d with a 0 value will force a download."
+					+ "Format for a Duration can be found here: https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#parse-java.lang.CharSequence-"
+					+ " Example: PT48H == \"48 hours\"")
+	private Duration fileAge;
+	
+	@Parameter(names = {"-c"}, description = "The path to the configuration file. Default is src/main/resources/config.properties")
+	private String configPath = "src/main/resources/config.properties";
```

Comment:
I'm not sure if it's possible, but if `@Parameter` can use a variable name, maybe swap these two lines and say `"Default is " + configPath` in case it needs to be updated so the doc and the code values stay synced?

Reply:
I think an annotation applies to the next thing declared after it, so I don't think swapping the order will work. It doesn't give a compiler error, but I think that's because the compiler assumes the annotation applies to the _next_ member variable listed.

Reply:
Ah okay.